nativelink-store/src/ontap_s3_store.rs

```diff
@@ -104,7 +105,7 @@ pub fn load_custom_certs(cert_path: &str) -> Result<Arc<ClientConfig>, Error> {
     );
 
     // Parse certificates
-    let certs = extract_certs(&mut cert_reader)
+    let certs = CertificateDer::pem_reader_iter(&mut cert_reader)
```

Comment:
is the error mapping still needed because of
`impl From<rustls_pki_types::pem::Error> for Error`